.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,33 @@
+name: CI Build
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
The name of the `master` branch` is `main`.

Reply:
Blarg

Reply:
Fixed.